packages/react/src/react/components/compound/auto-field/index.tsx

```diff
@@ -483,43 +484,18 @@ export function AutoField(props: AutoFieldProps) {
       )
     }
 
-    case 'create': {
-      if (!props.optionsFetchPath) throw new Error('Missing optionsFetchPath')
-
-      return (
-        <AutoRelationshipField
-          name={field.$client.fieldName}
-          fieldShape={field}
-          allowCreate={true}
-          allowConnect={false}
-          disabled={disabled}
-          optionsFetchPath={props.optionsFetchPath}
-        />
-      )
-    }
-    case 'connect': {
-      if (!props.optionsFetchPath) throw new Error('Missing optionsFetchPath')
-
-      return (
-        <AutoRelationshipField
-          name={field.$client.fieldName}
-          fieldShape={field}
-          allowConnect={true}
-          allowCreate={false}
-          disabled={disabled}
-          optionsFetchPath={props.optionsFetchPath}
-        />
-      )
-    }
+    case 'create':
+    case 'connect':
     case 'connectOrCreate': {
       if (!props.optionsFetchPath) throw new Error('Missing optionsFetchPath')
 
       return (
         <AutoRelationshipField
+          prefix={props.prefix || ''}
           name={field.$client.fieldName}
           fieldShape={field}
-          allowConnect={true}
           allowCreate={true}
+          allowConnect={false}
```

Comment:
3 options (connect, create, connectOrCreate) does not give the same result of `AutoRelationshipField`

It should be 

```tsx
allowConnect={field.type === "connect" || field.type === "connectOrCreate"}
allowCreate={field.type === "create" || field.type === "connectOrCreate"}
```